### DISCUSSION TITLE
Dopamine

### DISCUSSION THREAD

**Discussion by jayylmao** [catppuccin/catppuccin#1914] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the requested port be themed?

This is the app in question: https://github.com/digimezzo/dopamine. It comes with a themes folder with some alternate theme files (.theme) built-in. I based my port off of these.

### Has the requested port been themed already?

https://github.com/jayylmao/dopamine-catppuccin

**Comment by sgoudham** [comment]
Transferred over to #1917
